plugins/katello/3.4/release_notes/release_notes.md

```diff
@@ -4,12 +4,28 @@ title: Release Notes
 version: 3.4
 ---
 
-# Katello Nightly () Release Notes
+# Katello 3.4 (Oud bruin) Release Notes
 
-For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
+For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-3.4/CHANGELOG.md).
 
 ## Features
 
+#### File Repository Additions
+
+#### Repository Force Sync
+
+#### Content View Force Republish
+
+#### Subscription Updates
```

Comment:
@thomasmckay Would you mind writing up a release note re-cap of the various subscription updates (e.g. future dated subs, golden ticket, etc)?